Codeinwp/templates-patterns-collection
# Feat/ux improvements 101

### Summary
1. Resolved Double Scroll Bar on the Page Templates
2. Added Exit option for Import confirmation Modal.
3. Added back Icon on the `Back to starter sites` button, and rename the button to `Back to Starter Sites`
4. Fix spacing/punctuation on the `No results view`
5. Added skeleton loading for "Page Templates" and "My Library"

### Test instructions
1. Check that on import preview from "Page Templates" only one scroll bar is visible.
2. Check that after importing a page template, the final confirmation modal can be exited with `ESC` or click outside.
3. Check visual changes from 3, 4 and 5

Closes #101.
